### DISCUSSION TITLE
Stream at H264 Record at H265

### DISCUSSION THREAD

**Discussion by JustAnotherPU** [koush/scrypted#1507] [Ideas]
Hi Guys,

Since the new Unifi Protect update, H265 is finally supported, however upon activation it's breaks HomeKit compatibility as apple does not support H265 in HKSV. 
 
Can the protect plugin be updated to offer the possibility to grab the stream in H264 and still let Unifi protect store the recordings in H265 ? 

Thanks

**Comment by koush** [comment]
no

**Comment by JustAnotherPU** [comment]
too bad, guess I got to switch to homebridge and got2rtc
